internal/monitor/module_logger.go

```diff
@@ -0,0 +1,71 @@
+package monitor
+
+import (
+	"io/ioutil"
+
+	"github.com/lbryio/lbrytv/config"
+
+	"github.com/sirupsen/logrus"
+)
+
+// ModuleLogger contains module-specific logger details.
+type ModuleLogger struct {
+	ModuleName string
+	Logger     *logrus.Logger
+	Level      logrus.Level
+}
+
+// F can be supplied to ModuleLogger's Log function for providing additional log context.
+type F map[string]interface{}
+
+// NewModuleLogger creates a new ModuleLogger instance carrying module name
+// for later `Log()` calls.
+func NewModuleLogger(moduleName string) ModuleLogger {
+	logger := getBaseLogger()
+	l := ModuleLogger{
+		ModuleName: moduleName,
+		Logger:     logger,
+		Level:      logger.GetLevel(),
+	}
+	l.Logger.SetLevel(l.Level)
+	return l
+}
+
+// LogF is a deprecated method, an equivalent WithFields/WithField should be used.
+func (l ModuleLogger) LogF(fields F) *logrus.Entry { return l.WithFields(fields) }
```

Comment:
why deprecate it? this isn't a library. Its an internal repo. We should just remove it no?